GameData/RP-1/Contracts/Earth Observation 1/EOSCherenkov.cfg

```diff
@@ -73,23 +73,14 @@ CONTRACT_TYPE
 			type = Sequence
 			title = Survive in orbit for a day and transmit science
 
-			PARAMETER
-			{
-				name = HasPower
-				type = HasResource
-				resource = ElectricCharge
-				minQuantity = 1.0
-				title = Craft must have ElectricCharge after one day
-				hideChildren = true
-			}
-
 			PARAMETER
 			{
 				name = Orbit
 				type = Orbit
 				minEccentricity = 0.085

```

Comment:
If this is supposed to be elliptical (and go into the Van Allen belts) then the min ecc probably needs to be about 0.15 and the max more like 0.3 or so?

Reply:
Wait a minute, looking down later the science requirement is for space high, which means the min ecc has to be _way_ higher if the max Pe is 500km. You have to be able to _reach_ into spacehigh from a 200-500km Pe...

Reply:
Yeah, no clue where that came from.